app/views/layouts/_search_form.html.slim

```diff
@@ -0,0 +1,6 @@
+= form_tag url, method: :get, class: ['form-horizontal'] do
```

Comment:
@lowjoel I've to use form tag here, simple_form cannot create a form for me with out record.  And I also styled this to make my input field and search button in the same line. are you fine with it?